parlai/core/dict.py

```diff
@@ -690,6 +726,11 @@ def save(self, filename=None, append=False, sort=True):
         # save opt file
         with open(filename + '.opt', 'w', encoding='utf-8') as handle:
             json.dump(self.opt, handle, indent=4)
+        # save the byte level bpe model file as well
+        if self.tokenizer == 'bytelevelbpe':
+            self.byte_level_bpe.tokenizer.save(
+                os.path.dirname(filename), os.path.splitext(filename)[0]
```

Comment:
leave a comment saying "this puts it at filename.vocab and finlename.merges" or whatever